.meta/sinks/http.toml

```diff
@@ -1,7 +1,7 @@
 [sinks.http]
 batch_is_simple = true
 batch_size = 1049000
-batch_timeout = 5
+batch_timeout = 1
```

Comment:
I don't think you meant to change this? I don't see a corresponding code change?

Reply:
The existing docs were wrong: https://github.com/timberio/vector/blob/master/src/sinks/http.rs#L130

Reply:
Ah ok :+1: just wanted to make sure :)